microsoft-edge/webview2/concepts/developer-guide.md

```diff
@@ -0,0 +1,80 @@
+---
+description: Learn about best practices to use when developing your WebView2 application.
+title: WebView2 development best practices  
+author: MSEdgeTeam
+ms.author: msedgedevrel
+ms.date: 03/04/2021
+ms.topic: conceptual
+ms.prod: microsoft-edge
+ms.technology: webview
+keywords: WebView2, webview2, WebView, webview, edge, best practices  
+---
+# WebView2 development best practices  
+
+Every development team follows different practices when building their application. When you build WebView2 applications, there are best practices we recommend you follow. This article outlines those recommendations and best practices for you when building production based WebView2 applications.
+
+
+## Use Evergreen WebView2 Runtime  
+
+We generally recommend using the Evergreen WebView2 Runtime instead of the Fixed Version Runtime.  The Evergreen WebView2 Runtime updates automatically and includes the latest features and security patches available to your WebView2 application. The Evergreen WebView2 Runtime also requires less storage space on the disk.
+
+Ensure the Evergreen WebView2 Runtime is installed before using your WebView2 application.  For more information, navigate to [Deploying the Evergreen WebView2 Runtime][Webview2ConceptsDistributionDeployingEvergreenWebview2Runtime].  
+
+## Run regular compatibility tests
+
+When using the Evergreen WebView2 Runtime, ensure you run regular compatibility tests. Because the runtime is automatically updated, you'll want to ensure that your WebView2 application performs as expected.  For more information, navigate to [Stay compatible in Evergreen mode][Webview2ConceptsDistributionStayCompatibleEvergreenMode].  
+
+## Ensure APIs are supported by the installed WebView2 Runtime
+
+WebView2 applications need both a Webview2 SDK, and a WebView2 Runtime installed on the computer to run. Both the SDK and the runtime are versioned. Since APIs are continually being added to WebView2, new versions of the runtime are also released to support the new APIs. You'll need to ensure that the APIs used by your WebView2 application are supported by the WebView2 Runtime that's installed on the computer. 
+
+If you use the Evergreen WebView2 Runtime, consider the connectivity scenarios of your users. Some users may not be continually connected to the internet and may not use an updated runtime. When you push an update to your WebView2 application, the application may break because it uses newer APIs that are not available in the installed runtime. 
```

Comment:
```suggestion
If you use the Evergreen WebView2 Runtime, the runtime may not be on the latest version for limited scenarios. For example, if the user does not have internet access, then the runtime would not be automatically updated in that environment. Certain group policies may also pause WebView2 updates. When you push an update to your WebView2 application, the application may break because it uses newer APIs that are not available in the installed runtime. 
```